mam/v2/sponge.c

```diff
@@ -148,6 +148,41 @@ void sponge_absorbn(isponge *s, trit_t c2, size_t n, trits_t *Xs) {
   } while (0 < m);
 }
 
+void sponge_absorbn_flex(isponge *s, trit_t c2, size_t n, trit_array_t Xs[]) {
+  size_t total_trits = 0;
+  for (size_t i = 0; i < n; ++i) {
+    total_trits += Xs[i].num_trits;
+  }
+  trit_t buffer[total_trits + n];
```

Comment:
Why `+ n` ?

Reply:
oh, that's a remnant from a trial i did local due to ASAN failure due to `trytes_to_trits` , will remove